SignInCheckIn/SignInCheckIn/Services/ChildSigninService.cs

```diff
@@ -132,9 +132,13 @@ public ParticipantEventMapDto SigninParticipants(ParticipantEventMapDto particip
                     {
                         subItem.AssignedSecondaryRoomId = item.AssignedRoomId;
                         subItem.AssignedSecondaryRoomName = item.AssignedRoomName;
+                        subItem.CallNumber = item.CallNumber;
```

Comment:
This should only be the last 4 of the Event Participant ID currently it is null.